src/test/java/TestSuite.java

```diff
@@ -21,5 +24,11 @@ public static void setUp() {
         // This is also set in the CORSSpec, but if the whole TestSuite is executed,
         // this might be too late as a previously opened url connection might be re-used...
         System.setProperty("sun.net.http.allowRestrictedHeaders", "true");
+        TestHelper.setUp(TestSuite.class);
+    }
+
+    @AfterClass
+    public static void tearDown() {
+        TestHelper.tearDown(TestSuite.class);
```

Comment:
Why do we tear down here? Can't possible junit5 tests run in the same framework instance anymore?